numpy/ma/tests/test_core.py

```diff
@@ -759,6 +759,17 @@ def test_oddfeatures_3(self):
         atest[idx] = btest[idx]
         assert_equal(atest, [20])
 
+    def test_filled_with_basic_dtype(self):
```

Comment:
Could parametrize the test here, so that all the cases in the for loop will run even though one case fail.
```suggestion
    @pytest.mark.parametrize("dtype", [np.int8, np.uint8, np.complex64])
    def test_filled_with_basic_dtype(self, dtype):
```

Reply:
Thank you for your guidance! I wasn't very familiar with `pytest` before. I've made the changes based on your suggestions. Thanks again for your help!